### DISCUSSION TITLE
What mountain is nicknamed the "Savage Mountain"?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#293] [Galaxy Trivia] [answered]
**Hint:** also known as *Chogori*

**Comment by thuongtruong1009** [comment]
The mountain that is nicknamed the "Savage Mountain" is K2. K2 is the second-highest mountain in the world, with an elevation of 28,251 feet (8,611 meters), located on the border between Pakistan and China in the Karakoram Range. The nickname "Savage Mountain" is attributed to the extreme difficulty of ascending the peak, as well as the high number of fatalities that have occurred on the mountain.

**Comment by anggerpr** [comment, marked as answer]
K2 (8,611 meters/28,251 feet), or "Chogori," is the world's second highest mountain, after Everest (8,048 meters).  Also known as the "Savage Mountain," K2 is generally considered among the most difficult of the world's 8,000ers, and its death rate was historically one of the highest of all 14 mountains, along with Annapurna I (8,091 meters) and Nanga Parbat (8,126 meters). The former sits at a death rate of approximately 25%, while Parbat and K2 have traditionally held death rates in the low 20 percents.
